Plutarch/Internal.hs

```diff
@@ -144,7 +146,8 @@ hashRawTerm' (RConstant x) = addHashIndex 5 . flip hashUpdate (F.flat x)
 hashRawTerm' (RBuiltin x) = addHashIndex 6 . flip hashUpdate (F.flat x)
 hashRawTerm' RError = addHashIndex 7
 hashRawTerm' (RHoisted (HoistedTerm hash _)) = addHashIndex 8 . flip hashUpdate hash
-hashRawTerm' (RCompiled code) = addHashIndex 9 . flip hashUpdate (hashUTerm @alg code hashInit)
+hashRawTerm' (RPlaceHolder hash) = addHashIndex 9 . flip hashUpdate hash
+hashRawTerm' (RCompiled code) = addHashIndex 10 . flip hashUpdate (hashUTerm @alg code hashInit)
```

Comment:
I'd rather we kept existing hashing choices the same. It probably doesn't matter, but I'd rather the hash index of `RCompiled` stayed as `9`.